src/policy/ast.rs

```diff
@@ -215,6 +215,19 @@ impl<Pk: SimplicityKey> Policy<Pk> {
             _ => {}
         }
     }
+
+    /// Return an iterator over the fragments of the policy.
```

Comment:
In a036b0bc3a7930202e5ae0ed0a459f0c0bf36aeb:

You mention maybe using the dag module in the commit message -- actually we should use the `miniscript::iter` module which will be available in rust-miniscript 11.0. https://github.com/rust-bitcoin/rust-miniscript/pull/567

But for now it's fine  to have a manual implementation.

Reply:
I see. Miniscript's `iter` module is the cousin of Simplicity's `dag` module. I look forward to using it.